### PR TITLE
fix: convert without unit

### PR DESCRIPTION
**Pull request by boiawang**
fix [ref](https://github.com/alibaba/rax/issues/1575)

### CODE REVIEW

File: packages/stylesheet-loader/src/particular.js [hunk 1]
```diff
@@ -2,11 +2,10 @@
 
 import normalizeColor from './normalizeColor';
 
-const SUFFIX = 'rem';
-
 function convertUnit(val) {
-  if (/^\d+$/.test(val)) {
-    return val + SUFFIX;
+  // 判断数字
```
Comment (wssgcg1213) [inline packages/stylesheet-loader/src/particular.js]: English

File: packages/stylesheet-loader/src/particular.js [hunk 2]
```diff
@@ -2,11 +2,10 @@
...
+  if (/^[-+]?\d*\.?\d+$/.test(val)) {
```
Comment (wssgcg1213) [inline packages/stylesheet-loader/src/particular.js]: 这个正则对象定义到外面去吧

Comment (boiawang) [reply packages/stylesheet-loader/src/particular.js]: done

File: packages/stylesheet-loader/src/particular.js (hunk 1 above)
Comment (boiawang) [reply packages/stylesheet-loader/src/particular.js]: done

**Pull request merged**